src/main/java/be/seeseemelk/mockbukkit/command/ConsoleCommandSenderMock.java

```diff
@@ -236,14 +199,14 @@ public void sendRawMessage(@NotNull String message)
 	public void sendRawMessage(@Nullable UUID sender, @NotNull String message)
 	{
 		Preconditions.checkNotNull(message, "Message cannot be null");
+		System.out.println(ChatColor.stripColor(message));
```

Comment:
Im assuming this should be deleted 👀

Reply:
I was gonna leave it there since Paper logs it, but I suppose it's not really a big deal.